com.rawtherapee.RawTherapee.json

```diff
@@ -0,0 +1,268 @@
+{
+  "app-id": "com.rawtherapee.RawTherapee",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+  "sdk": "org.gnome.Sdk",
+  "command": "rawtherapee",
+  "rename-desktop-file": "rawtherapee.desktop",
+  "rename-icon": "rawtherapee",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc",
+    "--socket=x11",
+    /* Wayland */
+    "--socket=wayland",
+    /* Note playback */
+    "--socket=pulseaudio",
+    /* MIDI */
+    "--device=all",
+    /* Allow loading, saving files from anywhere (portals don’t work yet) */
+    "--filesystem=home",
+    /* Allow other instances to see lockfiles */
+    "--env=TMPDIR=/var/tmp"
```

Comment:
Did you just copy it from MuseScore?
https://github.com/flathub/org.musescore.MuseScore/blob/41b8fb9fbcb9d2ab83732975488ce5bac08d2448/org.musescore.MuseScore.yaml#L14-L30

Reply:
I copied from the other I did which may have come from MuseScore.